src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkInterfaceTests.ps1

```diff
@@ -887,3 +887,98 @@ function Test-NetworkInterfaceWithAcceleratedNetworking
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Test creating new NetworkInterfaceTapConfiguration using minimal set of parameters
+#>
+function Test-NetworkInterfaceTapConfigurationCRUD
```

Comment:
add a test for loadbalancer case as well

Reply:
I’m waiting for karthik to give me a sample Json. We have tested it manually. Fee free to open a bug to add this test later.